test/not_null.t.cpp

```diff
@@ -20,6 +20,24 @@
 
 using namespace gsl;
 
+#if gsl_HAVE( SHARED_PTR )
+using std::shared_ptr;
+using std::make_shared;
+#endif
+
+#if gsl_HAVE( UNIQUE_PTR )
+using std::unique_ptr;
+# if gsl_HAVE( MAKE_UNIQUE )
+using std::make_unique;
+# else
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args)
```

Comment:
Given that `make_unique<>()` was added only in C++14, perhaps it'd be a useful thing to have in `namespace gsl`, right next to the `unique_ptr<>` alias.

Reply:
I think it would be better to place it in the gsl::std14 namespace first, then import it to gsl next to std::unique_ptr. Also, I did not notice this part of the header, so I guess I have some unneeded usings in my test. :)